ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+### TODO: Move this to Dockerfile.upi.ci
+export HOME=/tmp
+pip install --user pyyaml
```

Comment:
Please don't incur a runtime cost with this, bake it into the image.

Reply:
Indeed. This was temporary while I determined what all needed to get baked into the image. It wasn't going to work anyhow as pip was not installed in the image. Already removed from my local repo.